website/src/components/Subscribe.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+class HubspotSubscribe extends React.Component {
+    public componentDidMount(): void {
+        const script = document.createElement('script')
+        script.src = '//js.hsforms.net/forms/v2.js'
+        document.body.appendChild(script)
+
+        script.addEventListener('load', () => {
+            ;(window as any).hbspt.forms.create({
+                portalId: '2762526',
+                formId: '9e9cd4d5-7a88-47e0-97e2-93dd11348ae5',
+                cssClass: 'subscribe-form',
+                target: '#hubspotSubscribeForm',
+            })
+        })
+    }
+    public render(): JSX.Element {
+        return (
+            <div id="newsletter-signup-footer" className="newsletter-form-footer row">
```

Comment:
Does this id need to exist?

Reply:
it would be good to keep for targeting in A/B tests.